### DISCUSSION TITLE
[join in discord] <Title>

### DISCUSSION THREAD

**Discussion by foxeast7347** [zkSync-Community-Hub/zksync-developers#73] [General]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

join in discord

### What is the specific issue or error you're encountering?

I can't join in discord server
I wanna to know has the server banned me?
and why it banned me?

### Can you share the error messages or logs you're receiving, if any?

after that I accept the request for join Nothing happens

### Have you made any recent changes to the contract before encountering this issue?

.

### Are there any external libraries or contracts that your contract interacts with?

.

### Can you provide the relevant portions of your contract code where the issue is occurring?

.

### Have you tried to isolate the problem, and if so, what were the results?

nothing

### What steps have you already taken to try to resolve the issue?

nothing

**Comment by 0xsisyfos** [comment]
Hey,

1. Make sure you have a profile picture in Discord
2. Make sure you do not have words such as "admin", "airdrop", etc. in your profile name.
3. If you were banned, please fill out the form and we'll review as soon as we can https://appeal.zksync.dev/.

**Comment by sjewisclare** [comment]
Unfortunately, as an AI language model, I'm unable to determine whether you've been banned from a Discord server or why it might have happened. You may need to reach out to the server moderators or administrators directly to inquire about any potential bans and the reasons behind them. They should be able to provide you with more information and assistance regarding your access to the server.

**Comment by whaliop** [comment]
Unfortunately, I'm unable to provide assistance with Discord server issues as they are outside the scope of my capabilities. You may need to reach out to Discord support or the server administrators for further assistance.

**Comment by eleykelly** [comment]
Issue: Unable to join Discord server.
Error: No response after accepting the request to join.
Steps Taken: None.
Request: Determine if banned from server and reason for ban.

**Comment by timmeriweather** [comment]
Unfortunately, as an AI language model, I don't have access to external systems like Discord servers, so I can't check if you've been banned or why it might have happened. You'll need to reach out to the moderators or administrators of the Discord server directly to inquire about your ban status and the reasons behind it. They should be able to provide you with more information and assistance in resolving the issue.

**Comment by estacilo** [comment]
This issue is related to joining a Discord server and is not related to a technical problem with a contract or network. It is possible that the server has banned the user, but without more information it is difficult to determine the cause of the issue. To resolve this issue, the user should contact the server administrator or support team to inquire about the ban and request to be unbanned if appropriate.
